feat(workers/branch): allow to define a blocked label (#12164)
renovatebot/renovate: docs/usage/configuration-options.md

<!-- markdownlint-enable MD001 -->

## stopRebasingLabel

> On supported platforms it is possible to add a label to a PR to request Renovate stop rebasing the PR branch after the first rebase.

Reviewer: ```suggestion
On supported platforms it is possible to add a label to a PR to request Renovate stop rebasing the PR.
```

<!-- markdownlint-enable MD001 -->

> ## stopRebasingLabel

Reviewer: ```suggestion
## stopUpdatingLabel
```

> On supported platforms it is possible to add a label to a PR to request Renovate stop rebasing the PR.
> By default this label is `"stop-rebasing"` however you can configure it to anything you want by changing this `stopRebasingLabel` field.

Reviewer: ```suggestion
On supported platforms it is possible to add a label to a PR to request Renovate stop updating the PR.
By default this label is `"stop-updating"` however you can configure it to anything you want by changing this `stopUpdatingLabel` field.
```
